src/services/admin.js

```diff
@@ -35,7 +35,7 @@ class NotificationHelper {
 		orgCode,
 		templateData = {},
 		subjectData = {},
-		tenantCodes,
+		tenantCode,
```

Comment:
This file still uses `tenantCodes` in a few places, which also seems off. Since we’re not doing any cross-tenant logic, that naming feels confusing. If there’s no real use case for `tenantCodes` in the code, can we rename everything to `tenantCode`?